src/main/java/domain/blackjack/BlackJack.java

```diff
@@ -0,0 +1,62 @@
+package domain.blackjack;
+
+import domain.participant.Dealer;
+import domain.participant.Participant;
+import domain.participant.Participants;
+
+import java.util.LinkedHashMap;
+
+public class BlackJack {
+
+    private final Dealer dealer;
+    private final Participants participants;
+
+    public BlackJack(final Dealer dealer, final Participants participants) {
+        this.dealer = dealer;
+        this.participants = participants;
+    }
+
+    public void beginDealing() {
+        dealer.receiveCard(dealer.draw());
+        dealer.receiveCard(dealer.draw());
+
+        for (Participant participant : participants.getValue()) {
+            participant.receiveCard(dealer.draw());
+            participant.receiveCard(dealer.draw());
+        }
+    }
+
+    public BlackJackResult saveParticipantResult() {
+        LinkedHashMap<Participant, WinStatus> result = new LinkedHashMap<>();
+        for (Participant participant : participants.getValue()) {
+            result.put(participant, isWinner(participant));
+        }
+        return new BlackJackResult(result);
+    }
+
+    WinStatus isWinner(Participant participant) {
+        if (!participant.canHit()) {
+            return WinStatus.from(false);
+        }
+        if (!dealer.canHit()) {
+            return WinStatus.from(true);
+        }
```

Comment:
개인적으로 WinStatus.from(boolean) 이 과정이 가독성을 해치는 것 같습니다.
명시적으로 지정해주면 어떨까요?
```suggestion
        if (!participant.canHit()) {
            return WinStatus.LOSE;
        }
        if (!dealer.canHit()) {
            return WinStatus.WIN;
        }
```

Reply:
넵 직접 명시해주는 것이 가독성을 높이겠네요.